### PR TITLE
Add demos for lab02.

### PR DESCRIPTION
**Pull request by gabrielmocanu**
Add demos for lab02.

Signed-off-by: Gabriel Mocanu <gabi.mocanu98@gmail.com>

### CODE REVIEW

**Comment by gabrielmocanu** [comment]
@razvanvirtan  Can you take a look on this, please?

**Review by razvanvirtan — APPROVED** [review]
Played with it on the IOCLA vm and everything looks good.

Reviewed-by: Răzvan Vîrtan <virtanrazvan@gmail.com>

File: laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+int main() {
+    
+    int a = 0;
+    int *p;
+    void *pv;
+
+    p = &a;
+    pv = &a;
+
+    printf("Before p = %p\n",p);
```
Comment (razvanvirtan) [inline laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c:16]: Just a minor thing here
```suggestion
    printf("Before p = %p\n",p);
```

File: laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    printf("Before &p = %p\n",&p);
```
Comment (razvanvirtan) [inline laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c:17]:
```suggestion
    printf("Before &p = %p\n", &p);
```

File: laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    p++;  
+    printf("After p = %p\n",p);
```
Comment (razvanvirtan) [inline laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c:19]:
```suggestion
    printf("After p = %p\n", p);
```

File: laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    printf("After &p = %p\n",&p);
```
Comment (razvanvirtan) [inline laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c:20]:
```suggestion
    printf("After &p = %p\n", &p);
```

File: laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    p++;
+
+    printf("Before pv = %p\n",pv);
```
Comment (razvanvirtan) [inline laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c:23]:
```suggestion
    printf("Before pv = %p\n", pv);
```

File: laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    pv++; 
+    printf("After pv = %p\n",pv);
```
Comment (razvanvirtan) [inline laborator/content/operatii-memorie-gdb/demo/pointers/test-pointers.c:25]:
```suggestion
    printf("After pv = %p\n", pv);
```

**Comment by RazvanN7** [comment]
@gabrielmocanu checkpatch.pl seems to be failing.

**Pull request merged**